examples/examples-gpu/nyc-taxi/rf-rapids-dask.ipynb

```diff
@@ -4,9 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p><code>list, SEE the cuml API docs</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>agree! fixed in https://github.com/saturncloud/examples/pull/58/commits/58a8c4dcce6b6821e95ff31d5bbd90a9e4abce46</p>